website/content/docs/integrations/consul/index.mdx

```diff
@@ -60,6 +60,19 @@ Consul data, Vault secrets, or just general configurations within a Nomad task.
 For more information on Nomad's template block and how it leverages Consul
 Template, please see the [`template` job specification documentation][].
 
+# DNS
+
+To provide Consul DNS to Nomad workloads using [bridge or CNI networking
+mode][], you will need to configure Consul's DNS listener to be exposed to the
+workload network namepsace. You can do this without exposing the Consul agent on
+a public IP by setting the Consul `bind_addr` to bind on a private IP address
+(the default is to use the `client_addr`).
+
+You will also need to set the [nameserver][] to the address. This address is
+exposed as the `consul.dns.addr` node attribute. Alternately, you can use Consul
```

Comment:
Not sure if I fully understand this part. 

Say my Consul agent is configured with `bind_addr = "10.0.0.2"`, then my job would need to look something like this?

```hcl
group "..." {
  network {
   mode = "bridge"
    dns {
      servers = ["10.0.0.2"]
    }
  }
}
```

Does the non-default port Consul uses for DNS affect this?

Reply:
Oh, that's a good point. Lemme double-check if you can give the nameserver a port and have that work (it'd have to be set to `consul.dns.port` in that case), but if not, then you need to bind Consul DNS to port 53 too.

Reply:
Nope, that doesn't work... I've fixed this section up and added a link to https://developer.hashicorp.com/consul/tutorials/networking/dns-forwarding which has a lot more detail.